src/scripts/breeding/Breeding.ts

```diff
@@ -20,27 +20,27 @@ class Breeding implements Feature {
     }
 
     initialize(): void {
-        this.hatchList[GameConstants.EggType.Fire] = [
+        this.hatchList[EggType.Fire] = [
             ['Charmander', 'Vulpix', 'Growlithe', 'Ponyta'],
             ['Cyndaquil', 'Slugma', 'Houndour', 'Magby'],
```

Comment:
With the `refactor/routes`, do you have any idea how we can move this multiple region stuff to world? It seems to make way more sense here but that would leave region-specific data fractured.

Reply:
We could change `hatchList = {}`
And when we initialise the region, just add it to `Breeding.hatchList[region] = this.hatchList`?

Feel like there is probably a better way, but at least that would keep all region data in 1 central location.